README.md

```diff
@@ -552,8 +643,24 @@ Training has experimental support for GPU acceleration with Nvidia CUDA or AMD R
 
 ```shell
 ilab model train --device=cuda
+
+```
+This version of `ilab model train` outputs brand-new models that can be served in the `~/.local/share/instructlab/checkpoints` directory.  These models can be run through `ilab model evaluate` to choose the best one.
+
+
+#### Train the model locally with multi-phase training and GPU acceleration
+
+`ilab model train` supports multi-phase training. This means in phase1 we train the model on knowledge, followed by evaluation which chooses the best checkpoint from that phase. The best checkpoint is then promoted to the starter model of phase2 which trains the model on skills. The skill phase is followed by evaluation in which the best checkpoint is chosen. Here is an example of how to run multi-phase training
+
+```shell
+ilab model train --strategy lab-multiphase --phased-phase1-data <KNOWLEDGE_TRAIN_MSGS_JSONL> --phased-phase2-data <SKILLS_TRAIN_MSGS_JSONL> -y`
```

Comment:
I recommend formatting the dataset placeholders in a way where it's clear that it's a file placeholder:

```suggestion
ilab model train --strategy lab-multiphase --phased-phase1-data <knowledge train messages jsonl> --phased-phase2-data <skills train messages jsonl> -y`
```